Assets/Mirror/Runtime/CustomAttributes.cs

```diff
@@ -10,51 +10,79 @@ public class NetworkSettingsAttribute : Attribute
         public float sendInterval = 0.1f;
     }
 
+    /// <summary>
+    /// Automatically syncs a value from the server to clients.
+    /// </summary>
     [AttributeUsage(AttributeTargets.Field)]
     public class SyncVarAttribute : Attribute
     {
+        ///<summary>A function that should be called when the value changes on the client.</summary>
         public string hook;
     }
 
+    /// <summary>
+    /// Command functions must start with 'Cmd' and they allow clients to send a command to the server to invoke a method.
```

Comment:
This is good,  but perhaps also explain What [Command] does?

Plenty of documentation you can take from here:
https://docs.unity3d.com/ScriptReference/Networking.CommandAttribute.html

same with other attributes